### DISCUSSION TITLE
Control Image Sensor Frame via Scripting Engine

### DISCUSSION THREAD

**Discussion by kopi23** [Stellarium/stellarium#2267] [Q&A]
Hi,

is it possible to control the Image Sensor Frame via Scripting Engine?
Meaning activating it, selecting a specific Sensor, Telescope, Lense and also the Rotation?

Thank you, kopi

**Comment by gzotti** [comment]
Please check the documentation: https://stellarium.org/doc/0.21/group__telescopeControl.html#classTelescopeControl.

**Reply by kopi23** [reply]
The description reads
_"Controlling a telescope" means receiving position information from the telescope and sending GOTO commands to the telescope._
I don't want to control a telescope. I'd like to display the rectangle which shows the FOV for a given Sensor/Lense combination.

**Reply by gzotti** [reply]
Oops, wrong plugin. https://stellarium.org/doc/0.21/group__oculars.html

**Reply by kopi23** [reply]
Thank you very much :)

**Reply by kopi23** [reply]
Hi again,
may I ask a follow up here?
I can now successfully activate the image sensor, select everything and rotate it.
I also move to a given RA/DEC with core.moveToRaDecJ2000(ra,dec).
But as time passes, the sensor rectangle drifts away from these coordinates. Is it possible to lock it at the set RA/DEC?
Thanks, kopi

**Reply by gzotti** [reply]
Switch to the other mount mode first.
https://stellarium.org/doc/0.21/classStelMainScriptAPI.html#a6e4de660cbb314ce9f1090cfabbd99a0

Or selectObjectByName first, then moveToSelectedObject.
